src/sql/workbench/api/node/sqlExtHost.protocol.ts

```diff
@@ -860,7 +860,6 @@ export interface ExtHostNotebookDocumentsAndEditorsShape {
 }
 
 export interface MainThreadNotebookDocumentsAndEditorsShape extends IDisposable {
-	$trySaveDocument(uri: UriComponents): Thenable<boolean>;
```

Comment:
None of these should be gone - how is stuff compiling?

Reply:
Good to know that I have to kill existing process and rebuild it again. Made changes.